README.md

```diff
@@ -7,6 +7,7 @@ An Ansible role for installing Open Trip Planner
 - `otp_data_dir` - Local directory to store OTP data (default: `/var/otp`)
 - `otp_repo` - Git repo to pull source from (default: `https://github.com/opentripplanner/OpenTripPlanner`)
 - `otp_version` - Commit to pull from (default: `339c1deb1daaa5edb63cbcd738318591ad9a61c7`, 0.11.x branch)
+- `otp_jarfile` - JAR file to run in daemon, absolute, or relative to `otp_bin_dir` (default: `./otp-core/target/otp.jar`, 0.11.x branch)
```

Comment:
Only concern with this is the absolute vs. relative handling. Does it make sense to use `otp_bin_dir` along with `otp_jarfile` to build the path vs. making it support absolute and relative?